examples/console/main.go

```diff
@@ -325,197 +323,238 @@ func run() error {
-		}
-		return nil
-	case "declarecloseidx":
-		idx, err := strconv.Atoi(params[1])
-		if err != nil {
-			return err
-		}
-		if idx >= len(closeAgreements) {
-			return fmt.Errorf("invalid index, got %d must be between %d and %d", idx, 0, len(closeAgreements)-1)
-		}
-		tx := closeAgreements[idx].SignedTransactions().Declaration
-		err = submitter.SubmitTx(tx)
-		if err != nil {
-			return err
-		}
-		return nil
-	case "closeidx":
-		idx, err := strconv.Atoi(params[1])
-		if err != nil {
-			return err
-		}
-		tx := closeAgreements[idx].SignedTransactions().Close
-		err = submitter.SubmitTx(tx)
-		if err != nil {
-			return err
-		}
-		return nil
-	case "deposit":
-		depositAmountStr := params[1]
-		destination := escrowAccount
-		if len(params) >= 3 && params[2] == "other" {
-			destination = otherEscrowAccount
-		}
-		account, err := horizonClient.AccountDetail(horizonclient.AccountRequest{AccountID: account.Address()})
-		if err != nil {
-			return fmt.Errorf("getting state of local escrow account: %w", err)
-		}
-		tx, err := txnbuild.NewTransaction(txnbuild.TransactionParams{
-			SourceAccount:        &account,
-			IncrementSequenceNum: true,
-			BaseFee:              txnbuild.MinBaseFee,
-			Timebounds:           txnbuild.NewTimeout(300),
-			Operations: []txnbuild.Operation{
-				&txnbuild.Payment{Destination: destination.Address(), Asset: asset.Asset(), Amount: depositAmountStr},
-			},
-		})
-		if err != nil {
-			return fmt.Errorf("building deposit payment tx: %w", err)
-		}
-		tx, err = tx.Sign(networkPassphrase, signer)
-		if err != nil {
-			return fmt.Errorf("signing deposit payment tx: %w", err)
-		}
-		_, err = horizonClient.SubmitTransaction(tx)
-		if err != nil {
-			return fmt.Errorf("submitting deposit payment tx: %w", err)
-		}
-		return nil
-	case "exit":
-		return errExit
-	default:
-		return fmt.Errorf("error: unrecognized command")
-	}
+		},
+	})
+	shell.AddCmd(&ishell.Cmd{
+		Name: "declarecloseidx",
+		Help: "declarecloseidx <idx> - declare to close the channel with a specific previous declaration tx",
+		Func: func(c *ishell.Context) {
+			idx, err := strconv.Atoi(c.Args[0])
+			if err != nil {
+				c.Err(err)
+				return
+			}
+			if idx >= len(closeAgreements) {
+				c.Err(fmt.Errorf("invalid index, got %d must be between %d and %d", idx, 0, len(closeAgreements)-1))
+				return
+			}
+			tx := closeAgreements[idx].SignedTransactions().Declaration
+			err = submitter.SubmitTx(tx)
+			if err != nil {
+				c.Err(err)
+				return
+			}
+		},
+	})
+	shell.AddCmd(&ishell.Cmd{
+		Name: "closeidx",
+		Help: "closeidx <idx> - close the channel with a specific previous close tx",
+		Func: func(c *ishell.Context) {
+			idx, err := strconv.Atoi(c.Args[0])
+			if err != nil {
+				c.Err(err)
+				return
+			}
+			tx := closeAgreements[idx].SignedTransactions().Close
+			err = submitter.SubmitTx(tx)
+			if err != nil {
+				c.Err(err)
+				return
+			}
+		},
+	})
+
+	shell.Run()
+	return
```

Comment:
```suggestion
```

Reply:
https://github.com/stellar/starlight/pull/403/commits/91ef9b5b287d67ecae82e8a540ac97f9bc7cd908

Reply:
forgot this one's actually needed
https://github.com/stellar/starlight/pull/403/commits/9c6ef4db4bc8bc357831b9f34e7182e8566ea48a